app/redux/reducers/todos.js

```diff
@@ -24,12 +24,37 @@ const completeTodoEntry = (state, action) => {
     }
 }
 
+const deleteTodoEntry = (state, action) => {
```

Comment:
You are only using `id` of `payload` from `action`.

You could destructure in the function definition.

```javascript
const deleteTodoEntry = (state, { payload: { id } }) => {
```